contrib/easyconfig-templates/2023a/PythonBundle-scipy.eb.tmpl

```diff
@@ -0,0 +1,39 @@
+# Template for a python bundle using SciPy-bundle on 2023a
+easyblock = 'PythonBundle'
+
+name = 'SoftwareName'
+version = '0.0.0'
+
+homepage = 'https://www.domain.org'
+description = """
+Description in 80 chars long column
+"""
+
+toolchain = {'name': 'gfbf', 'version': '2023a'}
+
+dependencies = [
+    ('Python', '3.11.3'),
+    ('SciPy-bundle', '2023.07'),
+]
+
+use_pip = True
+
+exts_list = [
+    ('ext-normal', 'ext_version', {
+        'checksums': ['0000000000000000000000000000000000000000000000000000000000000000'],
+    }),
+    ('ext-weird-name', 'ext_version', {
+        'modulename': 'import_name',
+        'checksums': ['0000000000000000000000000000000000000000000000000000000000000000'],
+    ('ext-wheel', 'ext_version', {
+        'source_tmpl': '%(name)s-%(version)s-py3-none-any.whl',
+        'checksums': ['0000000000000000000000000000000000000000000000000000000000000000'],
+    ('name-lower', 'ext_version', {
```

Comment:
can we add a comment here that 'name-lower' should be the name used on PyPI, i.e. the name you use when you `pip install` it?

also, 'ext_version' can be replaced with `version`

Reply:
We decided that we don't want comments in templates because they will probably stay in the final easyconfig.

So I tried to make this more clear without comments in https://github.com/easybuilders/easybuild-easyconfigs/pull/21984/commits/e43ce3d12109acce8aeaaffab5b194515bdaaa05